azkaban-web-server/build.gradle

```diff
@@ -50,27 +71,55 @@ sourceSets {
   }
 }
 
-bower {
-  installBase = 'bower' // <1>
+task movingJsTojsToPackage(dependsOn: ['npm_install']) {
 
-  'moment'('2.14.1'){
-    source 'min/moment.min.js' >> '/'
+  doLast {
+    copy {
+      from "node_modules/later/later.min.js"
+      into 'build/jsToPackage'
```

Comment:
yes. you are correct. That's the reason I pointed it out.
Symbols inside double quoted strings are resolved by gradle while single quoted string is not. This comes from Groovy and you can read more about it: http://groovy-lang.org/syntax.html#_double_quoted_string